src/bgg_companion_game_order.py

```diff
@@ -0,0 +1,47 @@
+from src.models.BoardGame import BoardGame
+
+
+class OrderBoardGames:
+    def __init__(self, board_games: list[BoardGame], order_by: str):
+        self.board_games = board_games
+        self.order_by = order_by
+
+    def order_games(self) -> list[BoardGame]:
+        if self.order_by == "alphabet":
```

Comment:
nit: maybe we can make these constants on the `OrderBoardGames` class? instead of having them as magic strings? so you can do

```
if self.order_by == OrderBoardGame.ORDED_BY_ALPHABET:
    # order by alphabet...
```

Reply:
Ahhhhhhh yes this looks much better, I don't know why I didn't see this.  To be honest the doing the conditional on those strings did look weird to me but just passed over it.  I like this much better thanks!